{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/conftest.py

```diff
@@ -1,16 +1,145 @@
 # pylint: disable=redefined-outer-name
+# flake8: noqa: E800
 import os
+from typing import Generator
 
 import pytest
-from playwright.sync_api import Playwright
+from django.db import connections
+from django.test.testcases import LiveServerThread, _StaticFilesHandler
+from django.test.utils import modify_settings
+from playwright.sync_api import BrowserContext, ConsoleMessage, Error, Page, Playwright
+from pytest_django.lazy_django import skip_if_no_django
 
 
 @pytest.fixture(scope="session")
-def playwright(playwright: Playwright) -> Playwright:
+def playwright(playwright: Playwright) -> Generator[Playwright, None, None]:
     """Override of playwright fixture so we can set up for use with Django.
 
     Background: https://github.com/microsoft/playwright-python/issues/439
     """
     os.environ["DJANGO_ALLOW_ASYNC_UNSAFE"] = "true"
 
-    return playwright
+    yield playwright
+
+
+@pytest.fixture
+def context(context: BrowserContext) -> Generator[BrowserContext, None, None]:
+    # Uncomment to disable or modify Playwright timeout
+    # context.set_default_timeout(0)
+
+    yield context
+
+
+@pytest.fixture
+def page(page: Page) -> Generator[Page, None, None]:
+    """Override of playwright page fixture that raises any console errors."""
+    page.on("console", raise_error)
+
+    yield page
+
+
+@pytest.fixture(scope="session")
+def live_server(request):
+    skip_if_no_django()
+    addr = (
+        request.config.getvalue("liveserver")
+        or os.getenv("DJANGO_LIVE_TEST_SERVER_ADDRESS")
+        or "localhost"
+    )
+
+    # Only way to fix static finding until this is merged:
+    # https://github.com/pytest-dev/pytest-django/pull/1032
+    server = LiveServer(addr)
+
+    yield server
+
+    server.stop()
+
+
+def raise_error(msg: ConsoleMessage) -> None:
+    """Raise an error if a console error occurs.
+
+    Args:
+        msg (ConsoleMessage): A console message.
+    Raises:
+        Error: An error message.
+    """
+    if msg.type != "error":
+        return
+
+    raise Error(f'error: {msg.text}, {msg.location["url"]}')
+
+
+class LiveServer:
```

Comment:
I thought this was handled by the library?

Reply:
It was, but I ran into the same issue I was having earlier with not being able to find static files. Either I wasn't able to serve the JS built assets, or I wasn't able to serve the stuff already in `static_source`.

I frankly wouldn't mind some pairing time on this, I still don't quite understand what exactly is going on, I just whacked away at it until it functioned. The final fix was adding in the `collectstatic` command before running the tests, so this code may no longer be necessary.